src/fhirtypes/StructureDefinition.ts

```diff
@@ -56,6 +56,11 @@ export class StructureDefinition {
    */
   elements: ElementDefinition[];
 
+  /**
+   * A base clone of the Structure Definition from before any rules were applied
+   */
+  _baseStructureDefinition: StructureDefinition;
```

Comment:
Since this is intended to be _private_ we should enforce that via the `private` accessor.